org-code-javabuilder/lib/src/main/java/dev/javabuilder/LocalMain.java

```diff
@@ -14,7 +11,9 @@
   public static void main(String[] args) {
     final LocalInputAdapter inputAdapter = new LocalInputAdapter();
     final LocalOutputAdapter outputAdapter = new LocalOutputAdapter(System.out);
-    final LocalProjectFileManager fileManager = new LocalProjectFileManager();
+    // replace with your testing url.
+    String url = "http://localhost-studio.code.org:3000/v3/sources/2sZQjPkr7-vE6zRhbEGTPg";
+    final UserProjectFileManager fileManager = new UserProjectFileManager(url);
```

Comment:
I'm working on a minimal websocket implementation of the local lambda function that is designed to work with dashboard. I'm trying to decide whether it makes sense to keep this local main function around. i.e. if we kept it, we'd use it to test javabuilder in isolation. Opinions?
Not blocking for this PR, but I bring it up for this reason: if we keep LocalMain, I think we'd want to keep it using the LocalProjectFileManager and have the websocket implementation be the one that reaches out to dashboard for its project.

Reply:
I don't have a strong preference beyond keeping the friction relatively low to testing locally. I can update LocalProjectFileManager to use the new parser instead. I found it much easier to call out to a url because it's a pain to format java code into a string manually, so I would rather have dashboard do it for me.

Reply:
👍 I'd say if you're going to keep using this pattern in your local testing, leave it as-is. If you're done, change it back. Once the websocket change is ready, the local websocket server will be the default way to do this.

Reply:
for multi-file, since we don't have it implemented on the javalab side yet I'll probably have to switch back to using local and will update LocalProjectFileManager to use the new file format. I can switch it back for now, although as of now LocalProjectFileManager does not exercise the new json parsing logic.

Reply:
switched back since I am done using this. In the follow up to this (support running multi-file) I'll update the LocalProjectFileManager to use a main.json instead of MyClass.java